services/api/src/resources/notification/sql.ts

```diff
@@ -33,7 +33,7 @@ export const Sql = {
         { type: type, content_type: contentType }
       )
       .where('nt.name', '=', name)
-      .select('nt.*', 'pn.*', knex.raw('? as type', [type]))
+      .select('nt.*', 'pn.*', knex.raw('? as orig_type', [type]))
```

Comment:
This looks like you're just clearing up a "confusing variable name" issue, but I don't see `orig_type` anywhere else so want to make sure

Reply:
@rocketeerbkw - it actually has to do with the fact that it was bombing out because of duplication of the name "type", which actually appears in project_notification. The two column names were causing issues with ambiguity.

Reply:
I'll double check this to make 100% sure though

Reply:
Yep - it's a bug in the existing query.